bin/j

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/env bash
+# Jira but as a quick and dirty CLI
+#
+# This script expects you to have a ~/.jira_credentials file with the following
+# contents:
+# ```yaml
+# user: "<your canonical email address>"
+# token: "<a valid Jira API token>"
+# board_id: <integer board id>
+# sprint_prefix: "<string prefix for you Jira pulse names>"  # e.g. "(Apps)"
+# ```
+#
+# See here for details of how to obtain a Jira API token:
+#   https://developer.atlassian.com/cloud/jira/platform/basic-auth-for-rest-apis/
+#
+# If you are adding anything to this then see here for API docs:
+#   https://developer.atlassian.com/cloud/jira/software/rest/intro/#introduction
+#   https://docs.atlassian.com/software/jira/docs/api/REST/9.17.0/#api/2/
+USER="$(grep user ~/.jira_credentials | cut -d' ' -f2 | tr -d '"')"
```

Comment:
Using something like seems safer:

```suggestion
USER="$(sed -n 's/^user:\s*"\([^"]\+\)"/\1/p' ~/.jira_credentials)"
```

Or at least use `-w` on grep not to catch other potential lines we don't want

Reply:
The code was changed accordingly